### DISCUSSION TITLE
Flameshot Plasma with X11 or Wayland

### DISCUSSION THREAD

**Discussion by RainerMathey** [flameshot-org/flameshot#3540] [Q&A]
... why did the autostart works properly under my "Plasma Wayland" session but when i start with "Plasma X11" it dosn't ???
I'm using:
Betriebssystem: EndeavourOS 
KDE-Plasma-Version: 6.0.2
KDE-Frameworks-Version: 6.0.0
Qt-Version: 6.6.2
Kernel-Version: 6.8.1-arch1-1 (64-bit)

**Comment by mmahmoudian** [comment]
Please fill a proper bug report and include all the information asked. This type of spamming our discussions and issues is inappropriate and not actionable.
